packages/patrol_cli/lib/src/commands/doctor.dart

```diff
@@ -34,35 +35,59 @@ class DoctorCommand extends PatrolCommand {
     return 0;
   }
 
+  void _printHeader() {
+    _logger.info('Patrol doctor:');
+  }
+
   void _printVersion() {
     _logger.info('Patrol CLI version: $version');
   }
 
   void _printAndroidSpecifics() {
-    _checkIfToolInstalled('adb');
+    _logger.info('Android: ');
+
+    _checkIfToolInstalled(
+      'adb',
+      'See the link: https://www.xda-developers.com/install-adb-windows-macos-linux/#how-to-set-up-adb',
+    );
 
     final androidHome = _platform.environment['ANDROID_HOME'];
     if (androidHome?.isNotEmpty ?? false) {
-      _logger.success('Env var \$ANDROID_HOME set to $androidHome');
+      _logger.success('• Env var \$ANDROID_HOME set to $androidHome');
     } else {
-      _logger.err(r'Env var $ANDROID_HOME is not set');
+      final linkHint = switch (_platform.operatingSystem) {
+        Platform.linux => 'https://stackoverflow.com/a/28889144/9899010',
+        Platform.macOS => 'https://stackoverflow.com/a/33631853/9899010',
```

Comment:
These instructions doesn't differ between OSes.

Moreover the macOS answer is outdated since it suggests to edit `.bashrc` (macOS uses `zsh` as default shell since 10.15 Catalina, so it should be `.zshrc`).